amundsen_application/static/js/components/common/SearchBar/InlineSearchResults/ResultItemList/ResultItem/index.tsx

```diff
@@ -0,0 +1,32 @@
+import * as React from 'react';
+import { Link } from 'react-router-dom';
+
+export interface ResultItemProps {
+  href: string
+  iconClass: string;
+  onItemSelect: () => void;
+  subtitle: string;
+  title: string;
+  type: string;
+}
+
+const ResultItem: React.SFC<ResultItemProps> = ({ href, iconClass, onItemSelect, subtitle, title, type }) => {
+  return (
+    <li className="list-group-item">
+      <Link className="result-item-link" onClick={onItemSelect} to={ href }>
+        <img className={`result-icon ${iconClass}`} />
+
+        <div className="result-info">
+          <div className="truncated">
```

Comment:
nit: do we need this extra wrapper or can it be merged with the `result-info` div above? Also it seems like we are very liberal about applying the `trucated` class to everything.

Reply:
I think so, I'm sure I couldn't get to this work as desired without all of these on here but I'll check again before merging to verify.